### PR TITLE
build: [ANDROAPP-5061] Maplibre integration

### PR DESCRIPTION
**Pull request by Balcan**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Review by ferdyrod — APPROVED** [review]

File: app/src/main/java/org/dhis2/metadata/usecases/MapStyleConfiguration.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package org.dhis2.metadata.usecases
+
+import org.dhis2.maps.layer.basemaps.BaseMapStyle
+import org.dhis2.maps.layer.basemaps.BaseMapStyleBuilder.build
+import org.hisp.dhis.android.core.D2
+import org.hisp.dhis.android.core.map.layer.MapLayerImageryProvider
+
+class MapStyleConfiguration(private val d2: D2) {
```
Comment (andresmr) [inline app/src/main/java/org/dhis2/metadata/usecases/MapStyleConfiguration.kt:8]: shouldn't this class be on maps module?

Comment (Balcan) [reply app/src/main/java/org/dhis2/metadata/usecases/MapStyleConfiguration.kt:8]: The map library has no knowledge of the sdk, that's why i have not include it.
But we could add the sdk dependency and move this class to the module if we think that is better

**Review by andresmr — APPROVED** [review]

**Comment by andresmr** [comment]
please, can you check CI?

**Review by andresmr — CHANGES_REQUESTED** [review]
CI is failing

**Pull request closed without merging**